end_to_end/test_convergence_1b_params_grain.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Let's avoid duplication -- the other one can do this via overrides, right?

Reply:
done